docker-compose.yml

```diff
@@ -362,6 +362,7 @@ services:
 
 networks:
   resources:
+    enable_ipv6: false
```

Comment:
Explicitly disabling IPv6 to reduce the number of variables in CI until we explicitly test it.